### PR TITLE
remove mailchimp vars

### PR DESCRIPTION
**Pull request by dannyroberts**
@snopoke FYI. No problems here, right? I'll also remove them from private variables if this is good to merge

### CODE REVIEW

**Comment by dannyroberts** [comment]
@proteusvacuum CC @calellowitz

File: ansible/roles/ansible-logrotate [hunk 1]
```diff
@@ -1 +1 @@
-Subproject commit 71b0ab39066f2dfd5b2701205cf398e97cda52c2
+Subproject commit 116cbc5517bfe65945844b61f7a317c1e349fe7c
```
Comment (czue) [inline ansible/roles/ansible-logrotate]: did you mean to include this change?

**Comment by snopoke** [comment]
no issues from me (except the submodule question)

File: ansible/roles/ansible-logrotate (hunk 1 above)
Comment (dannyroberts) [inline ansible/roles/ansible-logrotate]: Definitely not. I'll fix that, thanks!

**Comment by dannyroberts** [comment]
✅

**Pull request merged**